doc/mutations/creating-mutation-plugins.md

```diff
@@ -1,7 +1,192 @@
+
+```
+<?php
+namespace Drupal\graphql_examples\Plugin\GraphQL\Mutations;
+use Drupal\graphql\Annotation\GraphQLMutation;
+use Drupal\graphql\GraphQL\Execution\ResolveContext;
+use Drupal\graphql_core\Plugin\GraphQL\Mutations\Entity\UpdateEntityBase;
+use GraphQL\Type\Definition\ResolveInfo;
+/**
+ * Simple mutation for updating an existing article node.
+ *
+ * @GraphQLMutation(
+ *   id = "update_article",
+ *   entity_type = "node",
+ *   entity_bundle = "article",
+ *   secure = true,
+ *   name = "updateArticle",
+ *   type = "EntityCrudOutput!",
+ *   arguments = {
+ *     "id" = "String",
+ *     "input" = "ArticleInput"
+ *   }
+ * )
+ */
+class UpdateArticle extends UpdateEntityBase {
+  /**
+   * {@inheritdoc}
+   */
+  protected function extractEntityInput(
+    $value,
+    array $args,
+    ResolveContext $context,
+    ResolveInfo $info
+  ) {
+    return array_filter([
+      'title' => $args['input']['title'],
+      'body' => $args['input']['body'],
+    ]);
+  }
+}
+```
+
+The first thing we noticed is we are now using **UpdateEntityBase** instead of "CreateEntityBase" as our parent class,
+we can also see that we use the same argument "Input" as above but we also have another argument called **id**. The Graphql Module will be smart enough to use that id to match to the right entity.
+
+## Mutations to Delete Drupal Entities
+
+The only thing left now is really to delete the entity right? This is the simples type of operation out of the 3, because we only need to give graphql the **id**, it will check if we can access that type of operation and if so delete the entity with the id we give to it. So lets look at how the plugin looks like 
+
+```
+<?php
+namespace Drupal\graphql_examples\Plugin\GraphQL\Mutations;
+use Drupal\graphql_core\Plugin\GraphQL\Mutations\Entity\DeleteEntityBase;
+/**
+ * Simple mutation for deleting an article node.
+ *
+ * @GraphQLMutation(
+ *   id = "delete_article",
+ *   entity_type = "node",
+ *   entity_bundle = "article",
+ *   secure = true,
+ *   name = "deleteArticle",
+ *   type = "EntityCrudOutput!",
+ *   arguments = {
+ *     "id" = "String"
+ *   }
+ * )
+ */
+class DeleteArticle extends DeleteEntityBase {
+}
+```
+
+We can see that we use extend a "DeleteEntityBase" class and we only pass one argument, the id of the entity we want to delete, together with some of the anotations as we did previously with other mutations.
+
+## ArticleInput
+
+So we know from examples above that we need to define the arguments for mutations, much like a functions, mutations receive arguments that can be used to do whatever the mutation needs to do to work. In order for graphql to know information about the Arguments we create an "InputType". The ArticleInput that we used above looks like this : 
```

Comment:
"We know from the examples above that we need to define the arguments for mutations. Similar to how a function, mutations receive arguments

Reply:
Write "argument" lower case.

Make sure again to use `` back ticks to mark code blocks or names of classes, variables, etc.